examples/test_driven_software_defined_assets/consumption_datamart/assets/acme_lake/lake_io_manager.py

```diff
@@ -0,0 +1,26 @@
+from dagster import IOManager, io_manager
+from dagster_pandas import DataFrame
+
+
+class LakeIOManager(IOManager):
+
+    def __init__(self, datawarehouse_resource) -> None:
+        super().__init__()
+        self.datawarehouse_resource = datawarehouse_resource
+
+    def handle_output(self, context, obj: DataFrame):
+        pass
+
+    def load_input(self, context):
+        # TODO: Figure out a more elegant wat to pass the datawarehouse_resource to the type loader
```

Comment:
What do you get out of invoking a dagster type loader here vs. having the same logic inlined inside this function?

Something to think about in the implementation here is how this loading code knows the path to load from. I think that, in most cases, you'll want to use `context.asset_key` for this.